homeassistant/components/workday/binary_sensor.py

```diff
@@ -87,7 +87,7 @@ def setup_platform(
     discovery_info: DiscoveryInfoType | None = None,
 ) -> None:
     """Set up the Workday sensor."""
-    add_holidays: list[str] = config[CONF_ADD_HOLIDAYS]
+    add_holidays: list[date | datetime | str | float | int] = config[CONF_ADD_HOLIDAYS]
```

Comment:
According to schema it's a list of strings?

Reply:
We require it to be strings from the user but mypy requires it to match the type required by [`HolidayBase.append`](https://github.com/dr-prodigy/python-holidays/blob/390f59c386b15485c78550a6b4c017b7a6e09f67/holidays/holiday_base.py#L398) could probably tidy this up by importing `DateLike` from `holidays`